src/main/java/com/checkmarx/flow/service/SastScanner.java

```diff
@@ -378,7 +371,7 @@ private Integer handleUnKnownProjectId(Integer projectId, String ownerId, String
 
     private void checkScanSubmitEmailDelivery(ScanRequest scanRequest) {
         if (!ScanUtils.anyEmpty(scanRequest.getNamespace(), scanRequest.getRepoName(), scanRequest.getRepoUrl())) {
```

Comment:
org.apache.commons.lang3.StringUtils#isNoneEmpty can be used instead.

Reply:
Done.